modules/ROOT/pages/endpoint-as2-send.adoc

```diff
@@ -1,8 +1,8 @@
 = AS2 Send Endpoint Settings
 
-AS2 send endpoint settings configure the target endpoint in an outbound message flow to which transformed B2B messages are send to partners either directly or via third-party connections.
+Anypoint Partner Manager supports the AS2 Send to Partners endpoint type, which configures the target endpoint in an outbound message flow to which transformed B2B messages are sent to partners either directly or via third-party connections.
```

Comment:
Please change as:
Anypoint Partner Manager supports the AS2 Send to Partners endpoint type, which configures the target endpoint in an outbound message flow, or an acknowledgement endpoint in an inbound message flow, to which transformed B2B messages are sent to partners either directly or via third-party connections.